include/swift/ABI/MetadataValues.h

```diff
@@ -1979,16 +1978,19 @@ enum class TaskStatusRecordKind : uint8_t {
 
   /// A ChildTaskStatusRecord, which represents the potential for
   /// active child tasks.
-  ChildTask = 1,
+  ChildTask      = 1,
+  /// A GroupChildTaskStatusRecord, which represents a task group
+  /// and child tasks spawned within it (stored in similar fashion as ChildTask).
+  GroupChildTask = 2,
```

Comment:
Unless some strong reason to have cancellation as `2` thought that having the child types next to eachother was nicer